src/css/components/_non-media.scss

```diff
@@ -164,6 +164,9 @@
         max-width: 100%;
         max-height: 100%;
     }
+    div {
+        padding: 0;
+    }
```

Comment:
Seems that the following changes fix the issue without any noticeable issues (in `src/css/components/_v-scroll-menu-item.scss`):

```
    &__image {
        ...
        div {
            width: 100%;
-           height: 85%;
+           min-height: 85%;
            display: flex;
            flex-direction: column;
            align-items: center;
-           padding: 10px;
+           padding: 0;
```

No reason to be editing soft button image css for this